Pages/Table.php

```diff
@@ -228,7 +228,13 @@ public function __construct($options = array()) {
         if (isset($_POST['function'])) $this->function = $_POST['function'];
         if (isset($_REQUEST['id'])) $this->id = Request::get('id');
         if (isset($_REQUEST['p'])) $this->page_number = max(1, Request::get('p'));
-        $this->serial_update = Request::get('serialupdate', 'boolean');
+
+        /*
+         * serial_update can come as a GET parameter or with form POST processing,
+         * where it is a checkbox
+         */
+        $this->serial_update = (Request::get('serialupdate', 'boolean') OR (Request::get('serialupdate', 'string') == "on"));
```

Comment:
i think this should just check boolean for simplification - maybe update the request class to recognize "on" as true? it already recognizes "true" as a string:
https://github.com/macdabby/lightning/blob/master/Tools/Scrub.php#L228